test/utility/OrganizationEntity.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+import { Entity } from '../../es6/Entity.js';
+import { SimpleEntity } from '../../es6/SimpleEntity.js';
+
+export const classes = {
+	course: 'course',
+	active: 'active',
+	inactive: 'inactive'
+};
+
+export class OrganizationEntity extends Entity {
```

Comment:
we should probably just merge #11 at some point so we're testing actual code instead of artificial.  It's probably out of date by now though...

Reply:
I was wondering where those changes were - Since this is just testing the underlying Entity mocking the rest isn't an issue. Probably a name change would be better.